src/main/java/com/pillarglobal/miniApp/Calculator/controllers/services/CalculatorServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package com.pillarglobal.miniApp.Calculator.controllers.services;
+
+import com.pillarglobal.miniApp.Calculator.exceptions.MultipleOperatorException;
+import org.springframework.stereotype.Service;
+
+import java.util.Stack;
+
+@Service
+public class CalculatorServiceImpl implements CalculatorService {
+
+    public Integer calculate(String expression) throws MultipleOperatorException {
```

Comment:
Calculate does parsing? Why do we call it calculate then?

Reply:
I renamed it to solveExpression. Hope it's OK.

Reply:
Should we rename other things that have the word calculate in their name as well?

Reply:
Done.